providers/shared/services/external/id.ftl

```diff
@@ -1,4 +1,3 @@
 [#ftl]
 
 [#assign SHARED_EXTERNAL_RESOURCE_TYPE = "external" ]
-[#assign SHARED_EXTERNALNETWORK_RESOURCE_TYPE = "externalNetwork" ]
```

Comment:
Was this supposed to disappear?

Reply:
Yeah, didn't end up needed a resource to represent it